radixdlt/src/main/java/com/radixdlt/consensus/EpochManager.java

```diff
@@ -166,28 +183,54 @@ public void processEpochChange(EpochChange epochChange) {
 		queuedEvents.remove(nextEpoch);
 	}
 
+	private void processEndOfEpoch(VertexMetadata vertexMetadata) {
+		log.info("{}: END_OF_EPOCH: {}", this.loggerPrefix, vertexMetadata);
+		if (this.lastConstructed == null || this.lastConstructed.getEpoch() < vertexMetadata.getEpoch()) {
+			this.lastConstructed = vertexMetadata;
+
+			// Stop processing new events if end of epoch
+			// but keep VertexStore alive to help others in vertex syncing
+			this.bftEventProcessor = EmptyBFTEventProcessor.INSTANCE;
+		}
+	}
+
 	public void processGetEpochRequest(GetEpochRequest request) {
+		log.info("{}: GET_EPOCH_REQUEST: {}", this.loggerPrefix, request);
```

Comment:
This should be at debug or trace level, as it's in response to an external request and we don't have control over the rate.